pkg/controller/controller_instance.go

```diff
@@ -604,24 +661,42 @@ func (c *controller) updateInstanceCondition(
 	t := time.Now()
 
 	if len(instance.Status.Conditions) == 0 {
-		glog.Infof(`Setting lastTransitionTime for Instance "%v/%v" condition %q to %v`, instance.Namespace, instance.Name, conditionType, t)
+		glog.V(3).Infof(`Setting lastTransitionTime for Instance "%v/%v" condition %q to %v`, instance.Namespace, instance.Name, conditionType, t)
 		newCondition.LastTransitionTime = metav1.NewTime(t)
-		toUpdate.Status.Conditions = []v1alpha1.InstanceCondition{newCondition}
+		instance.Status.Conditions = []v1alpha1.InstanceCondition{newCondition}
 	} else {
 		for i, cond := range instance.Status.Conditions {
 			if cond.Type == conditionType {
 				if cond.Status != newCondition.Status {
-					glog.Infof(`Found status change for Instance "%v/%v" condition %q: %q -> %q; setting lastTransitionTime to %v`, instance.Namespace, instance.Name, conditionType, cond.Status, status, t)
+					glog.V(3).Infof(`Found status change for Instance "%v/%v" condition %q: %q -> %q; setting lastTransitionTime to %v`, instance.Namespace, instance.Name, conditionType, cond.Status, status, t)
```

Comment:
Doesn't upstream kube framework tweak the default settings? 
No action required.